cmake/herbert_FindMatlab.cmake

```
@@ -17,13 +17,17 @@ the path to the directory containing Matlab's dynamically linked libraries
 ``Matlab_UT_LIBRARY``
 the path to the Matlab UT library
 
+``Matlab_VERSION``
+the Matlab release e.g. 2019b
```

This will have the `R` preceding the year.

```suggestion
the Matlab release e.g. R2019b
```